docs-website/docusaurus.config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
       },
       items: [
         {
-          to: "docs/",
+          to: "docs/features",
```

Comment:
there's a number of other pages where we link to the main docs/ page

it'd be easier to remove the existing page, and change the slug for "docs/features" to be "docs/"

however, I think we need to give the docs/features page a slight UI uplift before elevating it - it just looks slightly less crisp than the current docs/ main page